gcn_email_service/helpers.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import threading
+
+
+# Adapted from https://gist.github.com/Depado/7925679
+logger = logging.getLogger(__name__)
+
+def periodic_task(interval):
```

Comment:
Fix PEP8 white space.

Also, it's the function, not the logger, that is adapted from the gist.

```suggestion
import threading

logger = logging.getLogger(__name__)


# Adapted from https://gist.github.com/Depado/7925679
def periodic_task(interval):
```